<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBookInfo</class>
 <widget class="QDialog" name="DBookInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书信息查询</string>
  </property>
  <widget class="QCheckBox" name="cbByName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>按书名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbByPress">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>按出版社</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePress">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbByAuthor">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>按作者</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAuthor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbQuery">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReset">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbShowAll">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>显示所有</string>
   </property>
  </widget>
  <widget class="QTableView" name="BookList">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>110</y>
     <width>715</width>
     <height>326</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ISBN</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbISBNList">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbQueryByISBN">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>70</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>定位</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbShowAll</sender>
   <signal>clicked()</signal>
   <receiver>DBookInfo</receiver>
   <slot>showAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbReset</sender>
   <signal>clicked()</signal>
   <receiver>DBookInfo</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbByName</sender>
   <signal>toggled(bool)</signal>
   <receiver>DBookInfo</receiver>
   <slot>setEnable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbByPress</sender>
   <signal>toggled(bool)</signal>
   <receiver>DBookInfo</receiver>
   <slot>setEnable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbByAuthor</sender>
   <signal>toggled(bool)</signal>
   <receiver>DBookInfo</receiver>
   <slot>setEnable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbQuery</sender>
   <signal>clicked()</signal>
   <receiver>DBookInfo</receiver>
   <slot>query()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbQueryByISBN</sender>
   <signal>clicked()</signal>
   <receiver>DBookInfo</receiver>
   <slot>queryByISBN()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showAll()</slot>
  <slot>reset()</slot>
  <slot>query()</slot>
  <slot>queryByISBN()</slot>
  <slot>setEnable()</slot>
 </slots>
</ui>
